Common/src/main/java/at/petrak/hexcasting/common/casting/actions/eval/OpForEach.kt

```diff
@@ -17,14 +17,15 @@ object OpForEach : Action {
         if (stack.size < 2)
             throw MishapNotEnoughArgs(2, stack.size)
 
-        val instrs = stack.getList(stack.lastIndex - 1)
-        val datums = stack.getList(stack.lastIndex)
+
+        val instrs = stack.getList(stack.lastIndex - 1, stack.size)
+        val datums = stack.getList(stack.lastIndex, stack.size)
+
         stack.removeLastOrNull()
         stack.removeLastOrNull()
 
         val frame = FrameForEach(datums, instrs, null, mutableListOf())
         val image2 = image.withUsedOp().copy(stack = stack)
-
```

Comment:
nit: weird whitespace change. won't even show up in blame since deleted line so it doesn't matter but still